docs/guides/ultimate-guides/torrents/_category_.json

```
@@ -0,0 +1,8 @@
+{
+  "label": "Torrents",
+  "position": 2,
+  "link": {
+    "type": "generated-index",
+    "description": "Torrents are a popular way to share content online. This section will tell you absolutely everything about torrents"
```

[Nitpick] Hmm...saying torrents at the end seems redundant. Also, it could be more engaging. What makes Torrent the best content-sharing method in comparison to other techniques?  
```suggestion
    "description": "Torrents are a popular way to share content online. This section will tell you absolutely everything about torrents"
```

---

Updated the description. 

Looked this up on google:

> The standard Google description length is 150-160 characters. The maximum SEO meta description length can be up to 300 characters.

This new description I am uploading is 293 characters. Still open to change, I do say torrents again in the second sentence.

---

Great but I think it'd be best to hear **your** opinion on it and not so much Google's.